src/rpc/blockchain.cpp

```diff
@@ -604,6 +604,20 @@ static UniValue SoftForkDesc(const std::string &name, int version, CBlockIndex*
     return rv;
 }
 
+static UniValue BIP9SoftForkDesc(const std::string& name, const Consensus::Params& consensusParams, Consensus::DeploymentPos id)
```

Comment:
Style, oh, never mind: https://github.com/bitcoin/bitcoin/blob/master/src/.clang-format#L27

I see, I didn't thought about the name.